app/lib/nomis_client/reasonable_adjustments.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+module NomisClient
+  class ReasonableAdjustments < NomisClient::Base
+    class << self
+      def get(booking_id:, reasonable_adjustment_types:)
+        return [] unless booking_id.present? && reasonable_adjustment_types.present?
+
+        reasonable_adjustments_response = get_response(booking_id: booking_id, reasonable_adjustment_types: reasonable_adjustment_types)
+
+        reasonable_adjustments_response['reasonableAdjustments'].map do |reasonable_adjustment_attributes|
+          attributes_for(reasonable_adjustment_attributes)
+        end
+      end
+
+      def get_response(booking_id:, reasonable_adjustment_types:)
+        NomisClient::Base.get(
+          "/bookings/#{booking_id}/personal-care-needs?type=#{reasonable_adjustment_types}",
+        ).parsed
+      end
```

Comment:
I'm intentionally not handling any errors at this level, as I'd like them to bubble up to be handled in the PER NOMIS importer